bedrock/firefox/templates/firefox/sync.html

```diff
@@ -0,0 +1,70 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% from "macros-protocol.html" import hero, picto_card, feature_card with context %}
+
+{% block page_title %}{{ ftl('firefox-sync') }}{% endblock %}
+
+{% block page_desc %}{{ ftl('access-and-sync') }}{% endblock %}
+
+{% block page_css %}
+ {{ css_bundle('sync') }}
+{% endblock %}
+
+{% block content %}
+<div class="mzp-l-content">
```

Comment:
Same as for /quantum, the hero component should not be nested within another `mzp-l-content`